src/backend/web/local/blueprint.py

```diff
@@ -51,6 +54,88 @@ def bootstrap_post() -> Response:
     )
 
 
+@local_routes.route("/certs")
+def certs() -> Response:
+    certs = {
+        "1": "-----BEGIN CERTIFICATE-----\nMIIDYTCCAkmgAwIBAgIULae0OluSv2DbeUa8xfrHzBujSiAwDQYJKoZIhvcNAQEL\nBQAwKzEpMCcGA1UEAwwgdGVzdC5hcHBzcG90LmdzZXJ2aWNlYWNjb3VudC5jb20w\nIBcNMjEwOTI0MDIxNTAxWhgPOTk5OTEyMzEwMDAwMDBaMCsxKTAnBgNVBAMMIHRl\nc3QuYXBwc3BvdC5nc2VydmljZWFjY291bnQuY29tMIIBIjANBgkqhkiG9w0BAQEF\nAAOCAQ8AMIIBCgKCAQEAokZw4SrVbUv12ZMCie/fZeyvfGxui2jAAe1V5DINXHuu\nwrXrTMksqHK2bRoVaCR3ghi2r5qmS5ppMTEASceSTY6ZAqOJ5H5+1rXIqok9v+lY\ny3vZI2iwYit6jPCe7qslj3VMMut/fuJqrjRfMJzD/ZKzHJ+PibFuN8rEu5Vc87w3\n8frMDJMXZQ6utXLVWxbPWFryI2aLVLYPSHRiuYHtlcX5z+kqukxhObbl+MBthv2+\nmnQWgNCk0MR1hNkppyIFLIjxVZtM2ehuGxtCjd0EIbVFmcpuxCNjwW/1qhegIwtK\nWKKu8hdAws4p3aZACr5zyUIiDn9i7DB5mYTFFUnl3wIDAQABo3sweTAJBgNVHRME\nAjAAMCwGCWCGSAGG+EIBDQQfFh1PcGVuU1NMIEdlbmVyYXRlZCBDZXJ0aWZpY2F0\nZTAdBgNVHQ4EFgQUJcoH+5oRWYmw0HlAa3mWkuCzXTIwHwYDVR0jBBgwFoAUJcoH\n+5oRWYmw0HlAa3mWkuCzXTIwDQYJKoZIhvcNAQELBQADggEBAALtSKIY9j+1tsg9\nSe5R74gpZQrPXhzN9NwTLyaWC2fOlqdBojojk+fMmfxTEi/6GsOSu76V64bdniJc\nW2VXS4e+YikG8jZa3jQd4sbCkyTvtMpRB3it2rPE9frLfFPlNNQe8s0uQ1qFdQCd\njUsQ4VUU5+gbmtEEFPM1zKJryE2Z9Ho+6lOOGhLk1bkPImWwaYV01Ky4ZZzbY7/m\nkNkwyugp7grXvLUJyC+kgBRZcI6NirUnqhjym6N1LuOiK5PHl0xogwq48xCYIn5n\nsg+tAgae5drfqSv2R44Qv40c7QfWhYEibOvGLOwdEHFxPKeQCOjMnnxCZAMR69ex\n6xIwD6A=\n-----END CERTIFICATE-----",
```

Comment:
- [ ] We should probably look to load this from a local file or something that doesn't get shipped to prod